src/config.ts

```diff
@@ -0,0 +1,65 @@
+const hasDOM = typeof document !== 'undefined' && typeof window !== 'undefined';
+const scriptElement: HTMLScriptElement = hasDOM && document.currentScript;
+// @ts-ignore
+const globalConfig: Record<String, string | string[] | null> =
+	globalThis.Prism?.constructor?.name === 'Object' ? globalThis.Prism : {};
```

Comment:
```suggestion
const globalConfig: Record<string, string | string[] | null> =
```

Ignore my suggestion if using uppercase `String` was intentional.